### ISSUE TITLE
Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by AStaroverov** [vuejs/vue-devtools#35]
Sometimes I get this error and have some trace 
```
Uncaught RangeError: Maximum call stack size exceeded                   $.cof.js:3
module.exports	@	$.cof.js:3
module.exports	@	$.classof.js:13
toString	@	es6.object.to-string.js:8
exports.isPlainObject	@	lang.js:276
sanitize	@	backend.js:468
(anonymous function)	@	backend.js:471
forEach	@	(program):950
sanitize	@	backend.js:470
(anonymous function)	@	backend.js:471
forEach	@	(program):950
postMessage (async)		
sendMessageToBackend	@	proxy.js:65
EventImpl.dispatchToListener	@	extensions::event_bindings:387
publicClass.(anonymous function)	@	extensions::utils:94
EventImpl.dispatch_	@	extensions::event_bindings:371
EventImpl.dispatch	@	extensions::event_bindings:393
publicClass.(anonymous function)	@	extensions::utils:94
dispatchOnMessage	@	extensions::messaging:310

```

Vue Devtools 1.0.2

**Comment by yyx990803** [comment]
This is because your data contians circular references. Fixed in 1.0.3.

**Comment by AStaroverov** [comment]
thx
